src/muiThemeable.js

```diff
@@ -1,15 +1,14 @@
 import React from 'react';
-import DefaultRawTheme from './styles/raw-themes/light-raw-theme';
-import ThemeManager from './styles/theme-manager';
+import getMuiTheme from './styles/getMuiTheme';
 
 function getDisplayName(WrappedComponent) {
   return WrappedComponent.displayName || WrappedComponent.name || 'Component';
 }
 
 export default function muiThemeable(WrappedComponent) {
-  const MuiComponent = (props, {muiTheme = ThemeManager.getMuiTheme(DefaultRawTheme)}) => {
-    return <WrappedComponent {...props} muiTheme={muiTheme} />;
-  };
+  function MuiComponent(props, {muiTheme = getMuiTheme()}) {
```

Comment:
The `muiThemeable` function serves the same purpose as the `wrapWithConnect` function from react-redux (I actually used that project as an inspiration when creating this one).  The main difference between this implementation and connect is that connect accepts parameters, and returns a function that wraps the component, so:

```js
connect(...args)(OurComponent);

/* As opposed to... */
muiThemeable(OurComponent);
```
In #2509 I created a version of muiThemeable that behaves more like connect that accepts arguments for mapping a baseTheme to localized style/theme objects (objects that only have the style information that the component needs).